### ISSUE TITLE
Breadcrumbs a11y violations

### ISSUE THREAD

**Issue by hxltrhuxze** [Royal-Navy/design-system#1096]
- [ ] Ensures every id attribute value is unique

**Comment by thyhjwb6** [comment]
Closing as there are not currently issues with `Breadcrumbs`.
